docs/develop/terra-js/getting-started.mdx

```diff
@@ -111,18 +109,17 @@ const wallet = lcd.wallet(mk);
 Although this tutorial has you input your mnemonic directly, this practice should be avoided in production.
 For security reasons, it's better to store your mnemonic key data in your environment by using `process.env.SECRET_MNEMONIC` or `process.env.SECRET_PRIV_KEY`. This practice is more secure than a hard-coded string.
 
-
 </Admonition>
 
 4. Request testnet funds for your wallet by navigating to the [Terra faucet](https://faucet.terra.money) and inputting your wallet address. You'll need these funds to perform swaps and pay for gas fees. Once the funds are in your wallet, you’re ready to move on to the next step.
 
 ## 4. Find a contract address
 
-To find the contract address for a specific Terraswap pair, visit https://app.terraswap.io/
+To find the contract address for a specific Terraswap pair, visit https://app.terraswap.io/.
```

Comment:
Sometimes this will parse the period. A workaround is "(https://app.terraswap.io/)[https://app.terraswap.io/]."